scopecheck/scopecheck.go

```diff
@@ -0,0 +1,85 @@
+package scopecheck
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/taskcluster/taskcluster-cli/extpoints"
+	"github.com/taskcluster/taskcluster-client-go"
+	"github.com/taskcluster/taskcluster-client-go/auth"
+)
+
+func init() {
+	extpoints.Register("scope-check", scopecheck{})
+}
+
+type scopecheck struct{}
+
+func (scopecheck) ConfigOptions() map[string]extpoints.ConfigOption {
+	return nil
+}
+
+func (scopecheck) Summary() string {
+	return "Shows whether a given scope or set of scope satifies another."
+}
+
+func (scopecheck) Usage() string {
+	return `Usage:
+  taskcluster scope-check <scope1> satisfies <scope2>
+`
+}
+
+func expandScope(scope2 []string) (*auth.SetOfScopes, error) {
+
+	a := auth.New(&tcclient.Credentials{})
+	a.Authenticate = false
+
+	scopes := &auth.SetOfScopes{
+		Scopes: scope2,
+	}
+
+	resp, err := a.ExpandScopes(scopes)
+	if err != nil {
+		fmt.Printf("Error expanding scopes: %s\n", err)
+		return nil, err
+	}
+
+	return resp, err
+}
+
+func (scopecheck) Execute(context extpoints.Context) bool {
+	argv := context.Arguments
+	scope1 := argv["<scope1>"].(string)
+	if argv["satisfies"] == true {
```

Comment:
@jonasfj Well to enable use of syntax of a form like `taskcluster scope-check scope1 satisfies scope2` I will introduce logic of this form `satisfies =: argv["<scope1>"].(string)` but I will not use the satisfies variable anywhere in my remaining logic it is to just enable that syntax. so the compiler complains(variable that is not used) but if I just performed this `argv["satisfies"] == true` I avail the syntax and make the compiler happy.....could there be a cleaner way of me achieving this..than this way?..by the way thanks for the review and sorry for the back and forth...